Meadowlark-js/packages/meadowlark-core/src/metaed/MetaEdValidation.ts

```diff
@@ -77,7 +77,13 @@ export function validateQueryParametersAgainstSchema(
 ): string[] {
   // TODO: RND-67 removed the Joi-specific code to report invalid query parameters
   // RND-307 will restore as a part of understanding Ajv error metadata
+  const schema = { ...metaEdModel.data.meadowlark.jsonSchema };
+
+  // Need to relax the validation such that no fields are "required"
+  schema.required.length = 0;
+
   const valid: ValidateFunction = ajv.compile(metaEdModel.data.meadowlark.jsonSchema);
```

Comment:
I didn't read the fine print apparently, when I went looking for "deep copy" and was reminded about the object spread operator. 

Well, I suppose my tests still passed because I was modifying the real required array rather than a copy!